### PR TITLE
complete react-accordion-component

### PR DESCRIPTION
**Pull request by hoangnguyen9384**
Screen Shot 2020-12-08 at 11 06 34 PM
Screen Shot 2020-12-08 at 11 06 49 PM
Screen Shot 2020-12-08 at 11 07 06 PM
Screen Shot 2020-12-08 at 11 07 15 PM

### CODE REVIEW

File: react-accordion-component/src/accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <App title="Cascading Style Sheets">
+          <div className="content">
+            <p>Cascading Style Sheets (CSS) is a style sheet language used for describing the presentation of a document written in a markup language like HTML. CSS is a cornerstone technology of the World Wide Web alongside HTML and Javascript.</p>
+          </div>
+        </App>
+        <App title="Javascript">
+          <div className="content">
+            <p>avascript, often abbreviated as JS, is a high-level, interpreted programming language that conforms to the ECMAScript specification. JavaScript has curly-bracket syntax, dynamic typing, prototype-based object-orientation, and first-class functions.</p>
+          </div>
+        </App>
```
Comment (scbowler) [inline react-accordion-component/src/accordion.jsx:49]: This structure should be created dynamically. The topics data should be created as an array in the `index.jsx` file then passed to accordion via props.

File: react-accordion-component/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+
```
Comment (scbowler) [inline react-accordion-component/src/index.jsx:4]: Create the array of topics here

File: react-accordion-component/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+ReactDOM.render(
+  <Accordion />,
```
Comment (scbowler) [inline react-accordion-component/src/index.jsx:6]: Pass array of topics to accordion here via props

```JSX
<Accordion topics={topicsData} />
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Interesting solution, only issue is that the topics data needs to be in an array and passed into the accordion via props then the jsx needs to be dynamically created in the accordion by looping through the topics data. See below comments for more details. Let me know if you have any questions.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**